### PR TITLE
feat: image-input-component

### PR DESCRIPTION
**Pull request by mikesch803**
add image square with responsiveness and input with standard and error message

### CODE REVIEW

**Comment by shahbazibnafzal** [comment]
Looks good to me.

**Review by Maianki — APPROVED** [review]
Looks good to me.

File: components/components.css [hunk 1]
```diff
@@ -21,6 +23,7 @@
     --COLOR-DANGER:#7F1D1D;
     --COLOR-SUCCESS:#14532D;
     --COLOR-GREY:#7e818c;
+    --COLOR-RED:red;
```
Comment (arung86) [inline components/components.css:26]: why to mix RGB and color names, maintain consistency

File: index.html [hunk 2]
```diff
@@ -134,19 +134,40 @@ <h3>Peanut butter</h3>
...
+                999</span>
+            <div class="card-btns">
+                <button class="card-cart btn btn-primary">add to cart</button><button
+                    class="card-wishlist btn btn-secondary">wishlist</button>
             </div>
         </div>
+    </div>
 
+    <!-- image component -->
+    <h3>responsive image</h3>
```
Comment (arung86) [inline index.html:152]: go for review with simple or related changes, so revw can understand and find things which need your  attention

File: components/components.css (hunk 1 above)
Comment (mikesch803) [reply components/components.css:26]: thanks, i changed this

**Pull request merged**